services/blockchain-indexer/shared/messageProcessor.js

```diff
@@ -74,7 +121,25 @@ const newRoundProcessor = async () => {
 	Signals.get('newRound').dispatch(response);
 };
 
-const initEventsProcess = async () => {
+const initMessageProcessors = async () => {
+	logger.info(`Registering job processor for ${accountMessageQueue.name} message queue.`);
+	accountMessageQueue.process(async (job) => {
+		const { account } = job.data;
+		logger.debug(`Scheduling indexing for account with address: ${account.address}.`);
+		await addAccountToDirectUpdateQueue(account);
```

Comment:
May be we can use `accountAddrUpdateQueue` from `indexer/accountIndex.js` and add address to the queue. For example:
```
await accountAddrUpdateQueue.add({address: account.address});
```

Reply:
This queue is only responsible for adding an address to the db. Also, does not have the functionality of de-duplication.

Reply:
Applied.